pkg/spoke/controllers/statuscontroller/availablestatus_controller.go

```diff
@@ -103,12 +105,23 @@ func (c *AvailableStatusController) syncManifestWork(ctx context.Context, origin
 	needStatusUpdate := false
 	// handle status condition of manifests
 	for index, manifest := range manifestWork.Status.ResourceStatus.Manifests {
-		availableStatusCondition := buildAvailableStatusCondition(manifest.ResourceMeta, c.spokeDynamicClient)
+		obj, availableStatusCondition := buildAvailableStatusCondition(manifest.ResourceMeta, c.spokeDynamicClient)
 		newConditions := helper.MergeStatusConditions(manifest.Conditions, []metav1.Condition{availableStatusCondition})
-		if !reflect.DeepEqual(manifestWork.Status.ResourceStatus.Manifests[index].Conditions, newConditions) {
+		if !equality.Semantic.DeepEqual(manifestWork.Status.ResourceStatus.Manifests[index].Conditions, newConditions) {
 			manifestWork.Status.ResourceStatus.Manifests[index].Conditions = newConditions
 			needStatusUpdate = true
 		}
+
+		values, err := c.getFeedbackValues(manifest.ResourceMeta, obj, manifestWork.Spec.ManifestConfigs)
+		if err != nil {
+			//TODO Update condition heres
```

Comment:
bypassing the error will lead to unexpected value update. in the first stage, we should at least return the error and additionally either reflect the error in the conditions or send an event

Reply:
return error will fallback and requeue. However, since it is most likely a user setting error, retry will still return error..So I think adding a condition only makes more sense.

Reply:
`utilruntime.HandleError` is more appropriate for errors that are skipped.

Reply:
set a condition to reflect this